rest-server/core/src/main/scala/com/actionml/core/store/backends/MongoStore.scala

```diff
@@ -81,6 +81,8 @@ class MongoStorage(db: MongoDatabase, codecs: List[CodecProvider]) extends Store
 
   private val timeout = 5 seconds
   private def sync[A](f: => Future[A]): A = Await.result(f, timeout)
+
+  override def dbName: String = db.name
```

Comment:
This is ok if you never assume the dbname is the engine-id. There are cases when they are not the same but the dbname is always passed to the Store.